test/storage_test.go

```diff
@@ -327,10 +328,48 @@ var _ = Describe(
 								Expect(err).Should(HaveOccurred())
 							},
 						)
-
 					},
 				)
```

Comment:
Should we add a check in webhook for:
When non-default non-persistent (SkipWorkDirValidate: true) workdir path is given in service context and no volume (emptydir) is specified. In this case, server crashes with root user and init crashes with non-root user